plugins/action_update_review_status/src/updateReviews.ts

```diff
@@ -10,14 +11,16 @@ interface Review {
   reviewStatus: ReviewStatus
 }
 
-async function updateReviews(input: any, DBConnect: any) {
+async function updateReviews({ parameters, applicationData, DBConnect }: ActionPluginInput) {
```

Comment:
Isn't applicationData removed?

Reply:
No, it's part of the main input object (see the issue #282 description). The intention was to standardise the input shape for all plugins (although not all of them use all the fields)

Reply:
Thanks for explaining this, so we still use this applicationData to send the current data (updated on every action on the chain), right?

Reply:
Yes that's right, which is also why things like `applicationId` and other basic parameters are no longer compulsory to define in the template Action